packages/slice-machine/tests/server/custom-types/delete.test.ts

```diff
@@ -22,34 +25,45 @@ describe("Delete Custom Type files", () => {
     errors: {},
     query: { id: CUSTOM_TYPE_TO_DELETE },
   } as unknown as RequestWithEnv;
-  const warn = console.warn;
   const error = console.error;
+  let mockRmSync: jest.SpyInstance;
+  let mockWriteFileSync: jest.SpyInstance;
+  let mockReadFileSync: jest.SpyInstance;
+  let mockAccessSync: jest.SpyInstance;
   afterEach(() => {
     vol.reset();
     jest.restoreAllMocks();
   });
   beforeEach(() => {
-    console.warn = jest.fn();
     console.error = jest.fn();
-    // @ts-expect-error don't need the right type for mocking purposes
-    jest.spyOn(fs, "lstatSync").mockReturnValue(true);
+    mockRmSync = jest.spyOn(fs, "rmSync");
+    mockWriteFileSync = jest.spyOn(fs, "writeFileSync");
+    mockReadFileSync = jest.spyOn(fs, "readFileSync");
+    mockAccessSync = jest.spyOn(fs, "accessSync");
   });
   afterAll(() => {
-    console.warn = warn;
     console.error = error;
   });
 
-  it("should delete all the custom type files", async () => {
-    const mockRmSync = jest.spyOn(fs, "rmSync");
-    const mockWriteFileSync = jest.spyOn(fs, "writeFileSync");
-    const mockReadFileSync = jest.spyOn(fs, "readFileSync").mockReturnValue(
-      JSON.stringify({
+  it("should delete and update all the custom type files", async () => {
+    vol.fromJSON({
+      "/test/customtypes/unwanted-ct/index.json": JSON.stringify({
         _cts: { "unwanted-ct": { name: "foo" }, "ct-2": { name: "bar" } },
-      })
-    );
+      }),
+      "/test/.slicemachine/assets/customtypes/unwanted-ct/mocks.json":
+        JSON.stringify({
+          _cts: { "unwanted-ct": { name: "foo" }, "ct-2": { name: "bar" } },
+        }),
+      "/test/.slicemachine/mock-config.json": JSON.stringify({
+        _cts: { "unwanted-ct": { name: "foo" }, "ct-2": { name: "bar" } },
+      }),
+    });
 
     const result = await deleteCT(mockRequest);
 
+    expect(mockRmSync).toHaveBeenCalled();
+
+    expect(mockRmSync).toHaveBeenCalledTimes(2);
```

Comment:
Is it worth checking the file is actually gone? In addition or instead of checking the methods called.